planner/core/testdata/integration_suite_out.json

```diff
@@ -203,6 +203,25 @@
           "├─IndexRangeScan_6(Build) 1.00 cop[tikv] table:t, index:b(b) range:[2,2], keep order:false, stats:pseudo",
           "└─TableRowIDScan_7(Probe) 2.00 cop[tikv] table:t keep order:false, stats:pseudo"
         ]
+      },
+      {
+        "SQL": "explain select /*+ USE_INDEX_MERGE(t, primary) */ * from t where 1 or t.c",
+        "Plan": [
+          "IndexMerge_7 6666.67 root  ",
+          "├─TableFullScan_5(Build) 10000.00 cop[tikv] table:t keep order:false, stats:pseudo",
+          "├─TableRowIDScan_6(Build) 6666.67 cop[tikv] table:t range:[-inf,0), (0,+inf], keep order:false, stats:pseudo",
```

Comment:
should it be `TableRangeScan`?

Reply:
Yes, explain info for index merge looks wrong here, @LittleFall PTAL.

Reply:
If `isChildOfIndexLookUp` is true, `explain` will think it a `RowsIDScan`.

And after `indexMerge` builded, it will set its `tablePlan`'s `isChildOfIndexLookUp`.

Because `scan6` occur both in `partPlans` and in `tablePlan` of `IndexMerge7`,  it will occur twice in `explain` result.

Why make it occur twice in this pr? I think `tablePlan` of `IndexMerge` can be `nil`. Because there are nil handles of it in `common_plans.go` and `task.go`.